packages/form/src/form/FormSection/renderField.tsx

```diff
@@ -184,13 +185,12 @@ const renderField = (props: RenderFieldProps) => {
         const files = get(item, name)
         const { bucketName } = fieldProps
         return (
-          <FormSectionReadOnlyStack
+          <DownloadableLinksField
             label={label}
             sx={readOnlySx}
-            title={files}
+            files={files}
             fileProps={{
               bucketName,
-              isFiles: true,
             }}
```

Comment:
@hieudaongoc @clodal This may break other dashboards

Reply:
@al1xt This component was actually removed a few commits back, so all other dashboards are already breaking. I simply added it back and separated it from the `FormSectionReadonlyStack`

Reply:
@al1xt it was removed [here](https://github.com/gravis-os/gravis-os/commit/8021aba6ab2c187272906ef478fe303192951162#diff-ce7fece584c128b386e5b93c3bdf865fe484373917b023c1d5a30d615089dc67)

Reply:
Yeah i doubt it’d break too